### ISSUE TITLE
[feature request] allow clickable URI to load EC and pre-fil pay-to field

### ISSUE THREAD

**Issue by jkister** [Electron-Cash/Electron-Cash#1450]
a URI on a webpage like [bitcoincash:abcd1234](bitcoincash:abcd1234) can load EC on testnet and pre-fill the "Pay to" field on the Send tab with the uri's address.

same for bchtest on EC testnet.

**Comment by cculianu** [comment]
I'm not sure what you are asking. EC already can do this if you "installed" it properly.

On Linux: put the .desktop file in the right place
On MacOS: run from .app
On Windows: install -setup.exe

What exactly needs to be added?  You want `bchtest` to be supported too? (That isn't supported)..

**Comment by EchterAgo** [comment]
On Linux you can also use https://github.com/AppImage/appimaged with the AppImage.

**Comment by jkister** [comment]
> What exactly needs to be added? You want bchtest to be supported too? (That isn't supported)..

Ah!  I was only testing testnet, when it didnt work I didnt even try mainnet.  sorry.

**Comment by cculianu** [comment]
Yeah we could add testnet too tho.  We don't have `bchtest:` registered with the OS on any of the platforms.  That can be added -- although it may not work 100% of the time unless we're careful due to quirks on some platforms (looking at *you* macOS!).

It's not a bad suggestion tho to add bchtest...
